tests/processors/test_flow_matching.py

```diff
@@ -107,9 +107,12 @@ def test_flow_matching_processor(
 
     processor = FlowMatchingProcessor(
         backbone=TemporalUNetBackbone(
-            in_channels=n_steps_output * n_channels_out,
-            out_channels=n_steps_output * n_channels_out,
-            cond_channels=n_steps_input * n_channels_in,
+            in_channels=n_channels_out,
+            out_channels=n_channels_out,
+            cond_channels=n_channels_in,
+            n_steps_output=n_steps_output,
+            n_steps_input=n_steps_input,
+            temporal_method="attention",
```

Comment:
Perhaps we can add a `pytest.mark.parametrize` to this test to cover the `temporal_method=["none", "attention", "tcn"]?
```suggestion
            temporal_method=temporal_method,
```